docker-compose.yml

```diff
@@ -11,6 +11,9 @@ services:
       - devstack_default
     volumes:
       - designer_mysql:/var/lib/mysql
+    # Uncomment these lines to access the database from localhost
+    # ports:
+    #  - "3307:3306"
```

Comment:
What is this for?

Reply:
I can show later if you like, but it is a nice to have for connecting to the database from localhost with a query browser.